docs/recipes.md

```diff
@@ -107,6 +107,37 @@ for path in sorted(Path("src").rglob("*.py")):  # (1)
 8. Actually write to the magic file.
 9. We can even set the `edit_uri` on the pages.
 
+> NOTE:
+> If your `edit_uri` is the default value or you have overwritten, you might
```

Comment:
```suggestion
> If your `edit_uri` is the default value or if you have overwritten it, you might
```

Reply:
By the way, these two cases represent... all the cases? So maybe the first sentence (condition) could be dropped?

Reply:
When writing it I had in mind that you have to look out for it in both cases, didnt read back and realize that it covers all conditions, will change